src/avfoundation.cs

```diff
@@ -12186,4 +12186,46 @@ interface AVRouteDetector {
 		[Export ("multipleRoutesDetected")]
 		bool MultipleRoutesDetected { get; }
 	}
+
+	[NoTV, iOS (11,0), NoWatch, NoMac]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface AVCapturePhoto
+	{
+		[Export ("timestamp")]
+		CMTime Timestamp { get; }
+
+		[Export ("rawPhoto")]
+		bool RawPhoto { [Bind ("isRawPhoto")] get; }
+
+		[NullAllowed, Export ("pixelBuffer")]
+		CVPixelBuffer PixelBuffer { get; }
+
+		[NullAllowed, Export ("previewPixelBuffer")]
+		CVPixelBuffer PreviewPixelBuffer { get; }
+
+		[NullAllowed, Export ("embeddedThumbnailPhotoFormat")]
+		NSDictionary<NSString, NSObject> EmbeddedThumbnailPhotoFormat { get; }
```

Comment:
I am not entirely sold on this being the generic version of the NSDictionary since we might want to turn this into a StrongDictionary at some point seems that the possible values are here to be found https://developer.apple.com/documentation/avfoundation/avassetwriterinput/video_settings_dictionaries?language=objc maybe just `NSDictionary WeakEmbeddedThumbnailPhotoFormat { get; }` so it is easily wrapped by the StrongDictionary generator.

Reply:
@dalexsoto I've fixed this now (and Metadata as well).